packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -771,8 +771,9 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
         appearance="outline"
         slot="anchor"
         id="people-picker-input"
+        role="combobox"
         placeholder=${placeholder}
-        aria-label=${this.ariaLabel || placeholder}
+        aria-label=${this.ariaLabel || this.strings.selectContact || placeholder}
```

Comment:
This ordering ensures that placeholder is only every used if a developer overrides the selectContact string to be a falsy value. Is this intentional?

Reply:
Not intentional, changing this.